examples/library/library_example_test.go

```diff
@@ -36,6 +38,10 @@ g.from(start: 1993-02-16T00:00:00Z, stop: 1993-02-16T00:03:00Z, count: 5, fn: (n
 	}
 
 	alloc := &memory.Allocator{}
+
+	if p, ok := program.(lang.DependenciesAwareProgram); ok {
+		p.SetExecutorDependencies(execute.Dependencies{"interpreter": dependencies.NewDefaultDependencies()})
```

Comment:
we should be able to change this to executetest.NewTestExecuteDependencies() also